src/index.html

```diff
@@ -17,16 +17,24 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="big">
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      class="search-bar"
+      data-qa="small">
```

Comment:
The `<form>` tag is incorrectly self-closing. It should have a separate closing tag `</form>` after the input element to properly encapsulate its contents.